zerver/lib/webhooks/git.py

```diff
@@ -86,6 +88,10 @@ def get_pull_request_event_message(
         main_message += '\n' + CONTENT_MESSAGE_TEMPLATE.format(message=message)
     return main_message.rstrip()
 
+def get_ping_message(user_name, integration):
```

Comment:
Would it be a good idea to have 2 templates, one with and one without the username? Or should I manage that in the function itself?